concordium-std/src/test_infrastructure.rs

```diff
@@ -1457,6 +1463,18 @@ impl<State: Serial + DeserialWithState<TestStateApi> + StateClone<TestStateApi>>
         Ok(res)
     }
 
+    fn upgrade(&mut self, module: ModuleReference) -> UpgradeResult {
```

Comment:
Please add documentation here, as people will use the TestHost directly and thus might see these docs.
Could you also add some docs for the `commit_state` now that you are at it? 🙏 😊

Reply:
Rust will use the documentation from the doc comments on the trait, so I would prefer not to duplicate them.